Jenkinsfile

```diff
@@ -16,6 +16,7 @@ stage("Unit Test") {
         export LD_LIBRARY_PATH=/usr/local/cuda-10.0/lib64
         export MPLBACKEND=Agg
         export MXNET_CUDNN_AUTOTUNE_DEFAULT=0
+        pip install --force-reinstall --upgrade gluonnlp==0.8.1
```

Comment:
Unit tests must be ran without gluonnlp if it isn't part of the dependencies. This PR appears to have passed CI, but in reality will not work for users.

If we don't plan to use the existing gluonnlp code, we can remove all of the autogluon nlp code.

Otherwise we need to move all imports of gluonnlp to lazy initialization and disable the nlp test until we have a more thorough CI capable of running tests with different dependencies installed for different tests.

Reply:
I think we need to revise the import structure to avoid importing everything when call `import autogluon`. For the testing purpose, it may be fine to install all required dependencies in the test.

Reply:
Yes, I agree that the importing structure should be revised. This is planned as part of the modularization effort. Unfortunately a lot of the existing tutorials and docs won't work anymore if this is changed right now.

Reply:
@Innixma Revised every appearance to lazy. Let me see if the CI can pass.